src/templates/next/layouts/render.tsx

```diff
@@ -41,6 +42,8 @@ export const renderComponent = ({
       return <Callout {...component} />
     case "collectionCard":
       return <CollectionCard {...component} />
+    case "collectionSearch":
+      return <CollectionSearch {...component} />
```

Comment:
related to earlier comment, actually I'm not sure if we should expose such "useless standalone" components to `renderComponent` since it should only ever be used in the context of say the `CollectionLayout`